src/components/NewTodo/NewTodo.tsx

```diff
@@ -0,0 +1,115 @@
+import { useState } from 'react';
+import { User } from '../../types/User';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  users: User[];
+  onAdd: (todo: Omit<Todo, 'id'>) => void;
+};
+
+export const NewTodo: React.FC<Props> = ({ users, onAdd }) => {
+  const [title, setTitle] = useState('');
+  const [titleHasError, setTitleHasError] = useState(false);
+
+  const [userSelect, setUserSelect] = useState(0);
```

Comment:
The initial state of 'userSelect' is set to 0, which is not a valid user. Consider setting the initial state to null or undefined to avoid confusion or potential bugs.